dashboard/app/views/levels/_free_response.html.haml

```diff
@@ -1,17 +1,14 @@
 - level ||= @level
-- last_attempt ||= @last_attempt
+- last_attempt = @last_attempt unless local_assigns.has_key? :last_attempt
 - index ||= nil
 
 - if index # If an index was provided we're inside a LevelGroup.
   :javascript
-    window['level_#{index}'] = {
+    window.levelGroup.questions[#{level.id}] = {
```

Comment:
I originally named it `results` but Brendan suggested I call it something else: https://github.com/code-dot-org/code-dot-org/pull/7874#discussion_r59661346